### DISCUSSION TITLE
Deploying uWebsockets app with kubernetes over digital ocean #673

### DISCUSSION THREAD

**Discussion by Gabriel-rd2** [uNetworking/uWebSockets.js#674] [Q&A] [answered]
I've being trying to achieve this for some time, but for some reason my requests to the external ingress endpoint (that resolves to a uWebSockes app service) go unhandled, is there some caveat about this kind of deploy i should know?

**Comment by ghost** [comment, marked as answer]
Uws doesn't know whether it runs on a calculator or a space ship, it's all the same to it

**Reply by Gabriel-rd2** [reply]
It was a problem with my ingress configuration, i used the default 1-click config for Digital Ocean and its working now
